deploy/skaffold/Dockerfile.deps

```diff
@@ -19,29 +19,29 @@ FROM alpine:3.10 as download-kubectl
 ARG ARCH
 # Track default version installed by Google Cloud SDK: 358.0.0 moved to 1.20(.10)
 # https://cloud.google.com/sdk/docs/release-notes
-ENV KUBECTL_VERSION v1.20.10
+ENV KUBECTL_VERSION v1.20.13
 ENV KUBECTL_URL https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/${ARCH}/kubectl
 # SHAs at gs://kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/${ARCH}/
-COPY deploy/skaffold/digests/kubectl.${ARCH}.sha512 .
```

Comment:
Can you double if the cloud build triggers work correctly?

For cloud build job, the workspace is root of skaffold. The last time I checked, `Copy` failed

Reply:
Just saw your comment. We changed the release infra to not run `make deps`. Instead the `build_deps` image is now built every single time. 

See -> https://github.com/GoogleContainerTools/skaffold/blob/main/deploy/skaffold/Dockerfile#L16

Maybe we should remove the `make deps` target.

Reply:
Ah, ok.  I'll take this as an opportunity to merge the various deploy/skaffold/Dockerfile.* into a single with independent targets.